src/main/java/service/impl/InputDataValidatorImpl.java

```diff
@@ -0,0 +1,17 @@
+package service.impl;
+
+import java.util.List;
+import service.InputDataValidator;
+
+public class InputDataValidatorImpl implements InputDataValidator {
+    private static final String VALID_FORMAT = "[sprb],[a-zA-Z]+,[0-9]+";
```

Comment:
```suggestion
    private static final Pattern VALID_RECORD = Pattern.compile("[sprb],[a-zA-Z]+,[0-9]+");
```
This way it's gonna be better for the performance since constant creation of Pattern object inside of str.get(i).matches may be expensive